src/foam/i18n/ClientCacheTranslationService.js

```diff
@@ -61,6 +62,7 @@ foam.CLASS({
         } else {
           this.initLatch.resolve();
         }
+        this.loadTheme();
```

Comment:
initLatch shouldnt resolve till this completes since we would still be loading locales